crates/store/re_grpc_client/src/lib.rs

```diff
@@ -273,7 +279,7 @@ async fn stream_catalog_async(
     let store_id = StoreId::from_string(StoreKind::Recording, "catalog".to_owned());
 
     let store_info = StoreInfo {
-        application_id: ApplicationId::from("rerun_data_platform"),
+        application_id: ApplicationId::from("redap_catalog"),
```

Comment:
I came here to request that you make this same change -- you must have hit the same issue as me.

Reply:
yeah, same app id -> same blueprint :) Hopefully we soon get that sorted with ``GetStoreInfo``.